File: 160706/number/ruby/two.rb
# 2. 实现一个方法，传入一个数字n，顺序打印100到n
num = 120
def sort_print_num(num)
  if num < 100 
    100.downto num do |i|
      i
      p i
    end
  else
    for j in 100..num
      j
      p j
    end
  end
end

sort_print_num(num)

File: 160706/number/java/two.java
// 2. 实现一个方法，传入一个数字n，顺序打印100到n
public class two{
  public static void main(String[] args) {
    int num = 110;
    sortPrint(num);
  }

  public static void sortPrint(int num){
    if (num < 100) {
      for (int i = 100; i >= num; i --) {
        System.out.println(i);
      }
    }else{
      for (int j = 100; j <= num; j ++) {
        System.out.println(j);
      }
    }
  }
}